doc/tutorials/pixel_data/compressing.rst

```diff
@@ -197,31 +200,5 @@ method is required.
 
 Note that in this case we also needed to change the *Photometric
 Interpretation*, from the original value of ``'YBR_RCT'`` when the dataset
-was using *JPEG 2000 Lossless* compression to ``'RGB'`` after decompression
-and recompression using *RLE Lossless*.
-
-
-Compressing datasets in a directory with ``multiprocessing``
-------------------------------------------------------------
-
-Our final example is to show how the :mod:`multiprocessing` module can be
-used to compress all the datasets in a directory:
-
-.. code-block:: python
-
-    from pathlib import Path
-    from multiprocessing import Pool, cpu_count
-
-    from pydicom import dcmread
-    from pydicom.uid import RLELossless
-
-    def compress(path: Path) -> None:
-        ds = dcmread(path)
-        ds.compress(RLELossless, encoding_plugin="pylibjpeg")
-        ds.save_as(path, write_like_original=False)
-
-    if __name__ == "__main__":
-        path = Path().resolve()
-
-        with Pool(processes=cpu_count()) as pool:
-            pool.map(compress, path.glob('*.dcm'))
```

Comment:
Removed because accounting for the *Photometric Interpretation* is far too complicated for the example.